### PR TITLE
Chemistry expansion

### PR DESCRIPTION
**Pull request by AtomicFox556**
#### Summary
Content "Add extra chemical recipes"

#### Purpose of change
Improve the variety of scavenged chemicals, and add new ways to make existing ones.

#### Describe the solution
Added recipes for already-existing acetone, chloroform, and citric acid.

Added washing soda and baking soda as common household chemicals, and calcium chloride as a less common one (used as a de-icing agent), along with recipes to produce them.

Changed the Prussian blue recipe components to more closely reflect what it'd actually be made of.

Modified glass recipe to actually make use of washing soda, rather than an abstracted process to make it first. Bone meal was removed as a possible alternative to limestone or quicklime: I found no mention of anything like that used for glass making anywhere, and the PR in which this recipe was added (#9580) just mentions that it might be possible without giving any sources of information.

Added baking soda as a possible alternative to ammonia for making crack. Also changed the recipe to use BOIL 2 quality instead of CHEM 3, as apparently people do this even with a spoon and a lighter, rather than using actual chemistry equipment.

Replaced "generic" oxidizer power in aspirin recipe with acetic anhydride.

Added ketene lamp, a new electric tool used to make acetic anhydride from acetone and acetic acid.

#### Describe alternatives you've considered
None.

#### Testing
Launched the game, made sure that no errors happen.

#### Additional context
None.

### CODE REVIEW

File: data/json/recipes/chem/chemicals.json [hunk 1]
```diff
@@ -39,6 +39,159 @@
...
+    "result": "chem_acetone",
+    "charges": 2,
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "chemistry",
+    "difficulty": 4,
+    "time": "1 h",
+    "batch_time_factors": [ 80, 2 ],
+    "autolearn": true,
```
Comment (TheShadowFerret) [inline data/json/recipes/chem/chemicals.json:53]: I don't think any of these could be autolearn 🤔

*Does not apply to Innawoods

Comment (AtomicFox556) [reply data/json/recipes/chem/chemicals.json]: Done.

File: data/json/items/resources/chemicals.json [hunk 2]
```diff
@@ -33,6 +33,57 @@
...
+    "container": "box_small",
+    "material": [ "powder_nonflam" ],
+    "volume": "250 ml",
+    "weight": "2540 mg",
+    "ammo_type": "components",
+    "count": 250
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_baking_soda",
```
Comment (NetSysFire) [inline data/json/items/resources/chemicals.json:72]: Couldn't this be an alternative to yeast in many food recipes? Might be worthwhile to add later.

Comment (AtomicFox556) [reply data/json/items/resources/chemicals.json:72]: It's not a simple substitution like that, you'd also need to add some kind of acid to use baking soda for this.

**Pull request merged**